genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/SetUpJobAction.java

```diff
@@ -185,6 +176,16 @@ protected void executeStateActionCleanup(final ExecutionContext executionContext
         fileManifestService.stop();
     }
 
+    private void relocateAgentLogFile(final File jobDirectory) {
```

Comment:
Moving pieces of code around really makes it hard to review because I need to check line by line to see if something actually changed or the block was moved down as-is. 
This PR is full of such changes.
Could you please submit a PR that re-formats the entire codebase so we can stop having these changes mixed in?

Reply:
sure. commit added to front of commit.
Could you please ensure you run the formatter before you commit from now on so that it doesn't drift?

Reply:
👍